<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comtrade</class>
 <widget class="QMainWindow" name="Comtrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comtrade</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>稳态性能测试</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>501</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>源波形</string>
         </attribute>
         <widget class="QTabWidget" name="tabWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>781</width>
            <height>481</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Cfg文件</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_12">
           <attribute name="title">
            <string>通道选择</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_13">
           <attribute name="title">
            <string>波形预览</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>试品波形</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>920</x>
            <y>350</y>
            <width>61</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>幅值：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="amplitude_2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>相位：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="phase_2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>频率：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="frequency_2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTabWidget" name="tabWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>771</width>
            <height>481</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Cfg文件</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_10">
           <attribute name="title">
            <string>通道选择</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox5">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>9</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_11">
           <attribute name="title">
            <string>波形预览</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox6">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>报告</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>暂态性能测试</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>511</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>源波形</string>
         </attribute>
         <widget class="QTabWidget" name="tabWidget_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>781</width>
            <height>481</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_15">
           <attribute name="title">
            <string>Cfg文件</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox7">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_16">
           <attribute name="title">
            <string>通道选择</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox8">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_17">
           <attribute name="title">
            <string>波形预览</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox9">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>试品波形</string>
         </attribute>
         <widget class="QTabWidget" name="tabWidget_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>781</width>
            <height>481</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_18">
           <attribute name="title">
            <string>Cfg文件</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox10">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_19">
           <attribute name="title">
            <string>通道选择</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox11">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_20">
           <attribute name="title">
            <string>波形预览</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox12">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>461</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_14">
         <attribute name="title">
          <string>报告</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>771</width>
            <height>481</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
